tgui/README.md

```diff
@@ -4,23 +4,31 @@
 
 tgui is a robust user interface framework of /tg/station.
 
-tgui is very different from most UIs you will encounter in BYOND programming. It is heavily reliant on Javascript and web technologies as opposed to DM. If you are familiar with NanoUI (a library which can be found on almost every other SS13 codebase), tgui should be fairly easy to pick up.
+tgui is very different from most UIs you will encounter in BYOND programming. It
+is heavily reliant on Javascript and web technologies as opposed to DM. If you
+are familiar with NanoUI (a library which can be found on almost every other
+SS13 codebase), tgui should be fairly easy to pick up.
```

Comment:
I at one point felt that it was more convenient to not hard wrap MD files simply because soft wrap exists, and hard wrap messes up diffs on even small changes.